gtk/src/default/gtk-3.20/_tweaks.scss

```diff
@@ -330,13 +331,14 @@ notebook > header {
                   border-color: transparent;                
                 }
               }
-              &:hover:not(:checked) { color: $insensitive_fg_color; }
+              &:hover:not(:checked):not(:backdrop) { color: $insensitive_fg_color; }
             }
           }
         }
       }
     }
   }
+  &:focus { box-shadow: 0 0 0 3px $focus_border_color;}
```

Comment:
oh... uhm... this shouldn't have been commited since it doesnt work - will remove it